exercism/bracket-push/bracket-push.js

```diff
@@ -0,0 +1,37 @@
+var bracket = function() {
+  if(bracket == '{}'|| '[]' )
+    return true
+}
+
+
+count number of items in bracket
+  if bracket.length % 2 !== 0 , there is a phantom bracket
+
+compare outer brackets
+
+
+
+
+
+
+
+
+
+
+// create an array of brackets in order
```

Comment:
Why should there be a function to create an array of brackets instead of manually making one?